### PR TITLE
Adding attributes to span

### PR DESCRIPTION
**Pull request by lipoja**
[CLOUDDST-21023]

### CODE REVIEW

**Review by chandwanitulsi — APPROVED** [review]

File: iib/common/common.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            'podman': run_cmd(podman_version_cmd, exc_msg='Failed to get podman version.').strip(),
+            'buildah': run_cmd(
+                buildah_version_cmd, exc_msg='Failed to get buildah version.'
+            ).strip(),
+        }
+    except FileNotFoundError:
+        return {'opm': '', 'podman': '', 'buildah': ''}
+
+
+def get_binary_versions_text() -> str:
```
Comment (yashvardhannanavati) [inline iib/common/common.py:30]: It feels like we can get rid of this function as it's only formatting text, maybe add it to the above function?

File: iib/common/common.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
```
[the remark this answers is not in the document]
Comment (lipoja) [reply iib/common/common.py]: renamed

File: iib/common/common.py (hunk 1 above)
Comment (lipoja) [reply iib/common/common.py]: removed

File: iib/common/common_utils.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    try:
+        return {
+            'opm': run_cmd(opm_version_cmd, exc_msg='Failed to get opm version.').strip(),
+            'podman': run_cmd(podman_version_cmd, exc_msg='Failed to get podman version.').strip(),
+            'buildah': run_cmd(
+                buildah_version_cmd, exc_msg='Failed to get buildah version.'
+            ).strip(),
+        }
+    except FileNotFoundError:
+        return {'opm': '', 'podman': '', 'buildah': ''}
```
Comment (yashvardhannanavati) [inline iib/common/common_utils.py:27]: nitpick: maybe log the error too?

**Review by yashvardhannanavati — APPROVED** [review]

**Pull request merged**